CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
+## [6.0.0] - 2021-09-29
+### Changed
+- Modified sftp Key Exchange option to accept an array instead of a string, allowing multiple kex algorithms
+
+### Upgrade steps
+
+With v6.0.0, we changed the Auth creation options to accept an array of Key Exchange algorithms rather than a string. To update, change the syntax of the auth commands. 
+```
+"keyExchanges":"diffie-hellman-group-a256"
+```
+becomes 
+```
+"keyExchanges":["diffie-hellman-group-a256"]
+```
+
```

Comment:
Great... a couple of more changes are needed that I just thought of.  
* For a major version change, we nee do ensure that any reference to /v5/ is now /v6/ in all imports.  
* Update the main doc.go and it's companion README.md the installation section for v6: 
  * Update for the new go install vs go get (see https://golang.org/doc/go-get-install-deprecation).  Maybe it makes sense to have both pre- and post- go 1.17. 
  * Add your upgrade instructions below the Install section.  heading of "Upgrading", subheading "Upgrading from v5 to v6", or something like that.

Reply:
Updated all references to v5 -> v6 in https://github.com/C2FO/vfs/pull/102/commits/390aefbdaddd91d61d5dfbab8d158e6158adc079 and https://github.com/C2FO/vfs/pull/102/commits/81c5dbb6fc9d63b06ff9cab52affa1250081b795

Added doc suggestions in https://github.com/C2FO/vfs/pull/102/commits/f797d8e31133002ea87e8d01bbf2f14e4344fb9e